client/my-sites/people/people-list-section-header/test/index.jsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import { shallow } from 'enzyme';
+import { expect } from 'chai';
+import sinon from 'sinon';
+
+/**
+ * Internal dependencies
+ */
+import useFakeDom from 'test/helpers/use-fake-dom';
+import useMockery from 'test/helpers/use-mockery';
+
+describe( 'PeopleListSectionHeader', function() {
+	let configMock,
+		PeopleListSectionHeader,
+		SectionHeader;
+
+	useFakeDom();
+
+	useMockery( mockery => {
+		configMock = sinon.stub();
+		configMock.isEnabled = sinon.stub();
```

Comment:
you might just use `import { stub } from 'sinon'` unless there's some problem with that